src/bno055_driver.cpp

```diff
@@ -307,15 +309,19 @@ rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
   BNO055Driver::on_activate(const rclcpp_lifecycle::State &)
 {
   RCLCPP_INFO(get_logger(), "%s is called.", __func__);
+  bool use_magnetometer = get_parameter("use_magnetometer").get_value<bool>();
```

Comment:
The configuration needs to be locked in before `on_activate` is called. We should pull the parameter and store it locally during `on_configure`.

Reply:
done.